.github/workflows/call-build-linux-arm-packages.yml

```diff
@@ -57,9 +57,9 @@ jobs:
       - name: Create Tarball and SHA256sums
         run: |
           TAR_FILE_NAME=valkey-${{inputs.version}}-${{matrix.distro.platform}}-${{ matrix.distro.arch}}
-          mkdir -p $TAR_FILE_NAME/bin $TAR_FILE_NAME/share
-          cp -rfv src/valkey-* $TAR_FILE_NAME/bin
-          cp -v /home/runner/work/valkey/valkey/COPYING $TAR_FILE_NAME/share/LICENSE
+          mkdir -p "$TAR_FILE_NAME/bin" "$TAR_FILE_NAME/share"
+          rsync -av --exclude='*.c' --exclude='*.d' --exclude='*.o' src/valkey-* "$TAR_FILE_NAME/bin/"
+          cp -v /home/runner/work/valkey/valkey/COPYING "$TAR_FILE_NAME/share/LICENSE"
```

Comment:
Interesting (weird?) that we include COPYING as LICENSE in the container file.

Reply:
Isn't that a requirement of the 2nd clause?

Reply:
Yes, we need to include the LICENSE when distributing the binaries.

Reply:
Yes, but the funny thing is that we rename the file.